src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/LaboratoryInvestigationCodedResult.java

```diff
@@ -0,0 +1,71 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+/**
+ * Details of the coded result of a Laboratory Investigation.
+ * <pre>
+ * Example: RSL+CV+::375211000000108:921::Bowel cancer screening programme FOB test normal (finding)'
+ * </pre>
+ */
+@AllArgsConstructor
+@Builder
+@Getter
+public class LaboratoryInvestigationCodedResult extends Segment {
+
+    private static final String KEY = "RSL";
+    private static final String QUALIFIER = "CV";
+    public static final String KEY_QUALIFIER = KEY + PLUS_SEPARATOR + QUALIFIER;
+
+    @NonNull
+    private final String investigationResultCode;
+    @NonNull
+    private final CodingType investigationResultCodeType;
+    @NonNull
+    private final String investigationResultDescription;
```

Comment:
This is already a `LaboratoryInvestigationCodedResult` class - probably we can lose the prefixes and still have the relevant detail

```suggestion
    @NonNull
    private final String code;
    @NonNull
    private final CodingType codingType;
    @NonNull
    private final String description;
```